src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerAdditionService.java

```diff
@@ -68,8 +69,11 @@ private IndividualBeneficialOwnerAddition getIndividualBeneficialOwnerAddition(
             BeneficialOwnerIndividualDto bo) {
         var actionDate = bo.getStartDate();
         var ceasedDate = bo.getCeasedDate();
-        var serviceAddress = bo.getServiceAddress();
```

Comment:
Altertnatively the XML could have defaulted the sevice address if none was passed. If that's equivalent to what's here then ok with change.

Reply:
My understanding is that it'll only be that way if the flag is true meaning that it will always be that way 

If the way we're doing it is changed that would also mean redoing this PR https://github.com/companieshouse/overseas-entities-api/pull/295